backend/social_connections/models.py

```
@@ -166,6 +166,12 @@ class DiscordEarnedRoleAssignment(BaseModel):
     class Meta:
         db_table = 'social_connections_discord_earned_role_assignment'
         ordering = ['-created_at']
+        permissions = [
+            (
+                'run_discord_earned_role_assignment',
+                'Can run Discord earned role assignment',
+            ),
+        ]
```

Ruff (0.15.21)

[warning] Mutable default value for class attribute

(RUF012)